src/android/android-uvc.cpp

```diff
@@ -0,0 +1,598 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#ifdef RS2_USE_ANDROID_BACKEND
+
+
+#include "android-uvc.h"
```

Comment:
Given that there's an `android_uvc.h` as well, could we rename theses file to `android_uvc_device.{h|cpp}` so it matches the contained class and doesn't get hella confusing?

Reply:
Yup. 208791f45